lighthouse-core/test/results/sample_v2.json

```diff
@@ -2044,57 +2154,11 @@
       "title": "Avoid non-composited animations",
       "description": "Animations which are not composited can be janky and increase CLS. [Learn more](https://web.dev/non-composited-animations)",
       "score": null,
-      "scoreDisplayMode": "informative",
-      "displayValue": "1 animated element found",
+      "scoreDisplayMode": "notApplicable",
```

Comment:
problem?

Reply:
I'd say yes. Were these manually added before?

Reply:
Nope, not that I saw from the commit history of that audit / this file.

I did find this: https://github.com/GoogleChrome/lighthouse/pull/11474/files#diff-09cd5d246b94b3c67132390d8e55b17af0a48f52b7054f52949c61548891fd96R1938 (the items dissapeared in this PR and came back in the following revert)

idk what's going on here.

Reply:
I think I manually added this to `artifacts.json`: https://github.com/GoogleChrome/lighthouse/pull/11246/files#diff-fccf071866992ae51684ed3c5edc57deda156a9ef8bbf41caeeeb5ba8931d205